package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "jupyterlab-unfold",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Leaving it for a separate release commit

```suggestion
  "version": "0.1.2",
```